### DISCUSSION TITLE
Build on RHEL7?

### DISCUSSION THREAD

**Discussion by jbuseman-eng** [containers/podman#11278] [Q&A] [answered]
Would it be possible to build podman 3+ on RHEL7?

**Comment by eriksjolund** [comment, marked as answer]
Yes, I believe so.
I've created a GitHub Actions workflow that builds Podman by using a CentOS 7 container:
https://github.com/eriksjolund/build-podman


A few days ago I bumped up the Podman version to v3.3.0-rc3
https://github.com/eriksjolund/build-podman/blob/podman-v3.3/config/centos7.json

I haven't tested the executable yet, but it can be downloaded from
https://github.com/eriksjolund/build-podman/actions/runs/1140818839
